cmd/epp/epp.go

```diff
@@ -123,20 +128,68 @@ func main() {
 
 	// Check
 	start = time.Now()
-	if batch {
-		dc, err := c.CheckDomain(domains...)
-		logif(err)
-		printDCR(dc)
-	} else {
-		for _, domain := range domains {
-			dc, err := c.CheckDomain(domain)
+	cmd = strings.ToLower(cmd)
+	switch cmd {
+	case "check":
+		handleCheck(c, obj, objs, batch)
+	case "info":
+		handleInfo(c, obj, objs)
+	default:
+		log.Fatal("Unknown command:", cmd)
+	}
+	qdur := time.Since(start)
+
+	color.Fprintf(os.Stderr, "@{.}Query: %s Avg: %s\n", qdur, qdur/time.Duration(len(objs)))
```

Comment:
I’d kind of prefer if this was a subcommand, e.g. `epp check [args]` and `app info [args]`.

Reply:
I don't quite follow. Could you elaborate a couple more examples?  I was following the same pattern as the `check` command albeit without the `batch` argument:

`epp check domain1 domain2 ... domainN`

`epp info domain1`

Reply:
It’s been a while since I worked on this, but it looks like the argument is `epp -cmd <cmd>` and not `epp <cmd>`.